dashboard/app/models/pd/enrollment.rb

```diff
@@ -60,7 +60,7 @@ class Pd::Enrollment < ApplicationRecord
   validates_presence_of :email, unless: :deleted?
   validates_confirmation_of :email, unless: :deleted?
   validates_email_format_of :email, allow_blank: true
-  validates :email, uniqueness: {scope: :pd_workshop_id, message: 'already enrolled in workshop'}, unless: :deleted?
+  validates :email, uniqueness: {scope: :pd_workshop_id, message: 'already enrolled in workshop', case_sensitive: false}, unless: :deleted?
```

Comment:
Note that this represents a slight change in functionality; currently, PD Enrollment enforces email uniqueness with case sensitivity; that means that we consider `foo@example.com` and `FOO@example.com` to be distinct values, which for emails is probably not what we actually want.

This does run the risk of some existing enrollments becoming invalid, but [a query of production data](https://codedotorg.slack.com/archives/C02EEGWLHR8/p1658527247655989?thread_ts=1658521233.659879&cid=C02EEGWLHR8) indicates that there is exactly one enrollment with an uppercase letter in the email field, and it doesn't conflict with any other rows. See [this Jira ticket for more](https://codedotorg.atlassian.net/browse/PLAT-1924)

Reply:
Thanks for taking care of this @Hamms !